app/main.py

```diff
@@ -2,0 +1,10 @@
+def copy_file(command: str) -> None:
+    cmd, origin, dupe = command.split()
+
+    if cmd != "cp" or origin == dupe:
+        return
+    try:
+        with open(origin, "r") as source, open(dupe, "w") as copy:
+            copy.write(source.read())
+    except FileNotFoundError as e:
+        print(f"Error: {e}")
```

Comment:
Clean Code: Although it's not a requirement, consider logging the error instead of printing it. This can be helpful for debugging and monitoring.